compiler/rustc_smir/src/rustc_smir/convert/ty.rs

```diff
@@ -771,7 +771,6 @@ impl<'tcx> Stable<'tcx> for ty::RegionKind<'tcx> {
         use stable_mir::ty::{BoundRegion, EarlyParamRegion, RegionKind};
         match self {
             ty::ReEarlyParam(early_reg) => RegionKind::ReEarlyParam(EarlyParamRegion {
-                def_id: tables.region_def(early_reg.def_id),
```

Comment:
What's the process for making breaking changes to stable mir? Do I need to update a version number somewhere?

Reply:
We're still in the "just break it" phase. There will be a public announcement and bot help once that changes